### ISSUE TITLE
target_level_change crashes on multiplayer

### ISSUE THREAD

**Issue by eezstreet** [JACoders/OpenJK#918]
(*submitting on behalf of Maui*)

Reporting a bug? Please make sure you've given the following information - thanks!

**Operating system and version:** Reported occurring on Windows 10, 64 bit

**Is this for single player or multiplayer?** Multiplayer

**Description of the bug (and if possible, steps to reproduce the bug):** Triggering any kind of level change via target_level_change causes the server to crash with no error given. The easiest way to reproduce this is to go to hoth2 and noclip to the end of the map

**What did you expect to happen instead?** It to not crash.

**Comment by ensiform** [comment]
It was fixed back when the coop branch was worked on but it wasn't necessarily the right fix. I think we won't be fixing it for OpenJK mp.  As, stock gameplay doesn't use these but maybe someone wants to. Just my 2¢

**Comment by shinyquagsire23** [comment]
I feel like it wouldn't exactly hurt to hook it to a server map change, but at the same time it might be desirable to just ignore it entirely to at least prevent the crash.

**Comment by Razish** [comment]
Agreed with @shinyquagsire23 - but it would be best to ignore it, I think.
If we hook it up to `nextmap`, I can see it ruining SP maps by having people rush to the end as soon as they spawn in. Of course this can be solved with .ent files
If we ignore it, existing maps will behave like every other map. I think that's the friendliest default option.

**Comment by nunziotocci** [comment]
I think the ideal scenario (I think) would be:
- Player reaches end - kicked to spectator and can't rejoin
- Once all players have reached the end, *then* trigger `nextmap`.

*Note: I haven't touch the OpenJK code, this is just a thought. I do enjoy OpenJK though!*

**Comment by Razish** [comment]
Unable to reproduce on Linux 64 bit with gcc. Tried noclipping to the end of `hoth2` and `taspir1`.
Oddly enough, we haven't changed that part of the code: https://github.com/JACoders/OpenJK/blame/eba428e/codemp/game/g_target.c#L963
It wasn't changed in modbase either: https://github.com/Razish/jasdk_modbase/blame/master/src/game/g_target.c#L947
I did change it in JA++ at some point, tagged as an SP/co-op fix: https://github.com/Razish/japp/commit/c963b42#diff-dabb2f9c5aa9b71eeb991e316c617b5aR865

Waiting for somebody else to confirm before touching this.
